src/pages/cplusplus/terms-to-know-for-beginners/index.md

```diff
@@ -1,26 +1,26 @@
 ---
 title: IDE and Printing different text 
 ---
-# Introduction to an IDE and printing different text : 
+# Introduction to an IDE and printing different text: 
 
-* In the last article, some download links for software required for programming . Software like this is known as an IDE.
+* In the last article, there were some download links for software required used by programmers. Software, like this, is known as an IDE.

```

Comment:
~~software required used by programmers~~ software that is used by programmers

Reply:
Looks good.